core/controllers/admin.py

```diff
@@ -127,6 +138,30 @@ def get(self):
     @require_super_admin
     def post(self):
         """Handles POST requests."""
+        def check_and_update_config_role(old_list, new_list, role):
```

Comment:
ok. done.

I did that as function was going to be used only inside this handler. Is this a bad practice ?

No. The plan is to delete it after replacement of admin_usernames etc. lists (As a part of 3rd milestone)